client/src/actions/apartments.js

```diff
@@ -0,0 +1,7 @@
+import { APARTMENTS_REQUEST } from '../constants/apartments';
+
+export default function fetchApartmentsRequest() {
+  return {
+    type: APARTMENTS_REQUEST
+  };
+}
```

Comment:
Mostly just because I saw some of the [redux docs](http://redux.js.org/docs/advanced/AsyncActions.html) do it, but I think we could definitely switch to them. I'm cool either way! 😄